src/colorizer/CanvasUIOverlay.ts

```diff
@@ -42,18 +38,12 @@ const defaultStyleOptions: StyleOptions = {
 const defaultScaleBarOptions: ScaleBarOptions = {
   ...defaultStyleOptions,
   minWidthPx: 80,
-  visible: false,
-  unitsPerScreenPixel: 1,
-  units: "",
+  visible: true,
 };
 
 const defaultTimestampOptions: TimestampOptions = {
   ...defaultStyleOptions,
-  visible: false,
-  frameDurationSec: 1,
-  startTimeSec: 0,
-  maxTimeSec: 1,
-  currTimeSec: 0,
```

Comment:
These were how `ColorizeCanvas` originally passed information to `CanvasOverlay`. Since `CanvasOverlay` now extends it instead, it has access to all the metadata needed to calculate these.